.github/workflows/push.yml

```
@@ -0,0 +1,90 @@
+        type: string
+      migrations-public-tags:
+        description: Migrations Docker image tags for public registry
+        required: false
+        type: string
+
+permissions:
+  id-token: write
+  packages: write
+
+jobs:
+  push:
```

actionlint (1.7.7)

"steps" section is missing in job "push"

(syntax-check)